wpilibc/src/main/native/include/frc/simulation/SolenoidSim.h

```diff
@@ -18,9 +18,46 @@ class SolenoidSim {
   SolenoidSim(PneumaticsModuleType type, int channel);
   ~SolenoidSim() = default;
 
+  /**
+   * Returns the solenoid output.
+   * @return the solenoid output.
+   */
+  [[deprecated("Use IsOn or IsOff methods instead.")]]
   bool GetOutput() const;
+
+  /**
+   * Sets the solenoid output.
+   * @param output The new solenoid output.
+   */
+  [[deprecated("Use SetOn or SetOff methods instead.")]]
```

Comment:
This shouldn't be deprecated (as @rzblue said in a previous comment, only get should be deprecated, set shouldn't)

Reply:
Sorry must've missed that one